docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -236,50 +238,96 @@ Additional configuration is available for some cell types.
+| [Gauge](#gauge)                           | Cells can be displayed as a graphical gauge, with several different presentation types. You can set the [Gauge display mode](#gauge-display-mode) and the [Value display](#value-display) options. |
 | Data links                                | If you've configured data links, when the cell type is **Auto**, the cell text becomes clickable. If you change the cell type to **Data links**, the cell text reflects the titles of the configured data links. To control the application of data link text more granularly, use a **Cell option > Cell type > Data links** field override. |
-| [JSON View](#json-view)                   | Shows values formatted as code.                                                                                                                                                                                                                                                                                                               |
-| [Image](#image)                           | If the field value is an image URL or a base64 encoded image, the table displays the image.                                                                                                                                                                                                                                                   |
-| [Actions](#actions)                       | The cell displays a button that triggers a basic, unauthenticated API call when clicked.                                                                                                                                                                                                                                                      |
+| [JSON View](#json-view)                   | Shows values formatted as code. |
+| [Image](#image)                           | If the field value is an image URL or a base64 encoded image, the table displays the image. |
+| [Actions](#actions)                       | The cell displays a button that triggers a basic, unauthenticated API call when clicked. |
+<!-- prettier-ignore-end -->
+
+#### Auto
+
+This cell type automatically displays values with sensible defaults applied.
 
-##### Sparkline
+It has the following cell options:
+
+<!-- prettier-ignore-start -->
+| Option | Description |
+| ------ | ----------- |
+| Wrap text | <p>Toggle the **Wrap text** switch to wrap text in the cell that contains the longest content in your table. To wrap the text _in a specific column only_, use a **Fields with name** [field override](ref:field-override), select the **Cell options > Cell type** override property, and toggle on the **Wrap text** switch.</p><p>Text wrapping is in [public preview](https://grafana.com/docs/release-life-cycle/#public-preview), however, it’s available to use by default.</p> |
```

Comment:
(no edits need, just filling in some context for future):

> to wrap text in the cell that contains the longest content in your table

i think this is true for the current table but not for the one that'll be replacing it. the new one wraps text in all cells that need it based on their content and column widths, not just the longest one.

> Text wrapping is in [public preview]

this will be GA in the new table.